jee-agents-distribution/src/main/assembly/scripts/agentadmin

```diff
@@ -49,6 +55,6 @@ else
     AGENT_OPTS=
 fi
 
-AGENT_CLASSPATH=\${AGENT_HOME}/lib/openam-installtools-launcher-${openam.version}.jar:\${AGENT_HOME}/lib/commons-io-1.4.jar
+AGENT_CLASSPATH=${AGENT_HOME}/lib/openam-installtools-14.1.17-SNAPSHOT.jar:${AGENT_HOME}/lib/commons-io-1.4.jar:${AGENT_HOME}/locale:${AGENT_HOME}/lib/openssoclientsdk.jar:${AGENT_HOME}/lib/agent.jar
```

Comment:
please change "14.1.17-SNAPSHOT"-> "${openam.version}"

Reply:
ok changing it